client/src/pages/AdminResourceManager.jsx

```diff
@@ -48,6 +49,10 @@ const AdminResourceManager = (props: Props): React$Element<'div'> => {
           <TabPane tab="Manage Resources" key="4">
             <ResourceManager />
           </TabPane>
+          <TabPane tab="Map View Entry" key="5">
+            <MapViewEntry selected />
+            <MapViewEntry selected={false} />
+          </TabPane>
```

Comment:
Could we comment this out/delete it just to keep things from appearing that aren't "supposed" to be there? Commenting out is fine since ik you use it for testing